web/src/message_edit_history.ts

```diff
@@ -391,4 +392,26 @@ export function initialize(): void {
             messages_overlay_ui.activate_element(this, keyboard_handling_context);
         },
     );
+
+    $("body").on("click", "#message-history-overlay .message_edit_history_content", (e) => {
+        e.stopPropagation();
+        e.preventDefault();
+
+        if (
+            $(e.target).closest(".message-media-preview-image:not(.message_inline_video)").length >
+            0
```

Comment:
Should we just directly check for the image element? This will future proof us too if we support preview for new file types.